virtcontainers/device/config/config.go

```diff
@@ -87,12 +87,48 @@ type DeviceInfo struct {
 	DriverOptions map[string]string
 }
 
+// BlockDrive represents a block storage drive which may be used in case the storage
+// driver has an underlying block storage device.
+type BlockDrive struct {
+	// File is the path to the disk-image/device which will be used with this drive
+	File string
+
+	// Format of the drive
+	Format string
```

Comment:
What format is `Format`? :)

- Could this be a type instead?
- Could you give some examples in a comment maybe?

Reply:
This part is picked from original code, it's the specified block format when pass to qemu, I think "raw" is the only supported format currently, not sure if there's any plan to add more format.
I'll add some comments to address this.

Reply:
Possible future format values are `rbd`, `iscsi`, `nbd` etc. We can add a slice of `key`:`value` mappings to allow to pass extra options. That is not required for this PR though. We can add it later.